Lib/gettext.py

```diff
@@ -417,6 +417,8 @@ def _parse(self, fp):
                     item = b_item.decode().strip()
                     if not item:
                         continue
+                    if item.startswith('#-#-#-#-#') and item.endswith('#-#-#-#-#'):
```

Comment:
Maybe add a comment saying something like: ignore "#-#-#-#-# ... #-#-#-#-#" comments.

Reply:
did it.

Reply:
Thanks.